src/main/scala/firrtl/ir/Serializer.scala

```diff
@@ -10,6 +10,8 @@ object Serializer {
   val NewLine = '\n'
   val Indent = "  "
 
+  val version = "1.1.0"
```

Comment:
This would probably be better described as a product and if this was defined globally so this can be used in `parser/Listener.scala`:

```scala
case class Version(major: Int, minor: Int, patch:Int) extends scala.math.Ordered[Version] {
  override def serialize: String = ???
  override def compare(that: Version): Bool = ???
}
```

This would simplify some things later, e.g., exiting if the input version is too new. 

Also, should this be defined in IR.scala?

Reply:
I'm okay with it being in `IR.scala` but I'd like to avoid modifying anything in IR.scala if we can avoid it (to avoid backwards compatibility concerns with existing transforms).

Reply:
I've made a `Version` class in `Serializer.scala` that is similar to this. Instead of having a `compare` method it has an `incompatible` method, since I think compare should check the patch version, but we want to be able to accept larger versions as long as only the patch is larger. Let me know if this needs more changes.

Reply:
I think this is reasonable. If we need to do more work here later we can but this looks good enough to me.